libft/srcs/ft_hash/update_value.c

```diff
@@ -1,29 +1,19 @@
 #include <stdlib.h>
 #include "ft_deque.h"
 #include "ft_hash.h"
-#include "ft_string.h"
 
 // key exist in hash
 // free(key, pre-content), update new content
-void	update_content_of_key(t_hash *hash, char *key, void *content)
+void	update_content_of_key(char **key, \
+								void *content, \
+								t_deque_node *target_node)
```

Comment:
```suggestion
void	update_content_of_key(t_deque_node *target_node\
                                                                   char **key, \
								void *content)
```
set_to_table, find_keyなどと合わせて、targetを第一引数にする、などいかがでしょう！

Reply:
✅